setup.cfg

```
@@ -64,18 +64,38 @@ test =
     coverage
 tests =
     %(test)s
+dev =
+    black~=22.3
+    doc8~=0.10
+    flake8~=4.0
+    ipython
+    isort~=5.10
+    pre-commit
+    pre-commit-hooks
+    %(docs)s
+    %(test)s
 all =
     %(doc)s
     %(resmon)s
     %(popylar)s
     %(tests)s
+    %(dev)s
 
 [flake8]
 max-line-length = 99
 doctests = False
 exclude=
-    *build/
-    docs/
+    __pycache__,
+    .git,
+    .pytest_cache,
+    .tox,
+    build,
+    dist,
+    tests,
+    venv,
+    ci/templates,
+    .eggs,
+    docs
```

This was working fine, not sure it is necessary.
```suggestion
    *build/
    docs/
```